Projekty/Projekt1/Kacper_Grzymkowski_Jan_Gaska/Inzynieria Cech, wstępne modelowanie/WUM_Kacper_Grzymkowski_Jan_Gaska_Kamien_Milowy_II.ipynb

```diff
@@ -0,0 +1,2102 @@
+{
```

Comment:
<p>Ciekawe byłoby sprawdzenie jak zmiana sposobu enkodowania wpływa na łatwość modelowania (one-hot oraz target encoding). Jest to analiza w celach dydaktycznych</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>